config/locales/en.yml

```diff
@@ -54,6 +54,7 @@ en:
               link_and_identifier_exists: Enter either the website where users can apply for the licence or the licence identifier
             licence_transaction_licence_identifier:
               taken: This licence identifier is already in use
+              invalid: Enter a licence identifier. This is a 5 or 6 digit number
```

Comment:
Should this show the actual format (like: enter a license identifier in the format XXX-X-X or XXXX-X-X), and maybe something about where you get the id from?

Reply:
I had similar concerns, but me and Katherine agreed that the shorter prompt is probably fine. TL;DR: 
> I think the proposed message is probably fine considering the point that new codes / or possibly existing ones will need to be sourced from us anyway, and thus us explaining the exact structure still doesn’t help the user if they don’t know the exact code they need beforehand. I’ll make the change!

You can read the conversation in the [Trello card](https://trello.com/c/uFIvraIX/2288-validate-licence-identifer-format#comment-65d76482aa851240ec89ccbd).